src/Game/System/GalaxyCometScheduler.cpp

```diff
@@ -0,0 +1,456 @@
+    { "OceanRingGalaxy", COMET_CATEGORY_NORMAL },
+    { "ReverseKingdomGalaxy", COMET_CATEGORY_COIN_100 },
+    { "FactoryGalaxy", COMET_CATEGORY_NORMAL },
+    { "OceanRingGalaxy", COMET_CATEGORY_COIN_100 },
+    { "FactoryGalaxy", COMET_CATEGORY_NORMAL },
+    { "ReverseKingdomGalaxy", COMET_CATEGORY_NORMAL },
+    { "FactoryGalaxy", COMET_CATEGORY_COIN_100 },
+    { "OceanRingGalaxy", COMET_CATEGORY_NORMAL },
+    { "ReverseKingdomGalaxy", COMET_CATEGORY_NORMAL }, 
+    { "FactoryGalaxy", COMET_CATEGORY_COIN_100 },
+    { "OceanRingGalaxy", COMET_CATEGORY_NORMAL },
+    { "ReverseKingdomGalaxy", COMET_CATEGORY_COIN_100 },
+    { "FactoryGalaxy", COMET_CATEGORY_NORMAL },
+    { "ReverseKingdomGalaxy", COMET_CATEGORY_NORMAL },
+    { "FactoryGalaxy", COMET_CATEGORY_COIN_100 },
+    { "OceanRingGalaxy", COMET_CATEGORY_COIN_100 },
+    { "FactoryGalaxy", COMET_CATEGORY_NORMAL },
+    { "ReverseKingdomGalaxy", COMET_CATEGORY_COIN_100 },
+    { "OceanRingGalaxy", COMET_CATEGORY_NORMAL },
+    { "FactoryGalaxy", COMET_CATEGORY_COIN_100 },
+    { "OceanRingGalaxy", COMET_CATEGORY_COIN_100 },
+    { "ReverseKingdomGalaxy", COMET_CATEGORY_COIN_100 },
+    { nullptr, 0 },
+};
+
+const GalaxyCometTimePaper sCometTimeTableGrandGalaxy6[] = {
+    { "CannonFleetGalaxy", COMET_CATEGORY_COIN_100 },
+    { "OceanPhantomCaveGalaxy", COMET_CATEGORY_COIN_100 },
+    { "CannonFleetGalaxy", COMET_CATEGORY_NORMAL },
+    { "HellProminenceGalaxy", COMET_CATEGORY_NORMAL },
+    { "CannonFleetGalaxy", COMET_CATEGORY_COIN_100 },
+    { "HellProminenceGalaxy", COMET_CATEGORY_COIN_100 },
+    { "OceanPhantomCaveGalaxy", COMET_CATEGORY_NORMAL },
+    { "CannonFleetGalaxy", COMET_CATEGORY_COIN_100 },
+    { "OceanPhantomCaveGalaxy", COMET_CATEGORY_COIN_100 },
+    { "CannonFleetGalaxy", COMET_CATEGORY_COIN_100 },
+    { "HellProminenceGalaxy", COMET_CATEGORY_NORMAL },
+    { "OceanPhantomCaveGalaxy", COMET_CATEGORY_COIN_100 },
+    { "CannonFleetGalaxy", COMET_CATEGORY_NORMAL },
+    { "HellProminenceGalaxy", COMET_CATEGORY_COIN_100 },
+    { "OceanPhantomCaveGalaxy", COMET_CATEGORY_NORMAL },
+    { "HellProminenceGalaxy", COMET_CATEGORY_NORMAL },
+    { "CannonFleetGalaxy", COMET_CATEGORY_NORMAL },
+    { "OceanPhantomCaveGalaxy", COMET_CATEGORY_NORMAL },
+    { "HellProminenceGalaxy", COMET_CATEGORY_COIN_100 },
+    { "CannonFleetGalaxy", COMET_CATEGORY_NORMAL },
+    { "HellProminenceGalaxy", COMET_CATEGORY_NORMAL },
+    { "OceanPhantomCaveGalaxy", COMET_CATEGORY_COIN_100 },
+    { "HellProminenceGalaxy", COMET_CATEGORY_COIN_100 },
+    { "OceanPhantomCaveGalaxy", COMET_CATEGORY_NORMAL },
+    { nullptr, 0 },
+};
+
+const GalaxyCometTimePaper * const sGalaxyCometTimerPaper[] = {
+    sCometTimeTableGrandGalaxy1,
+    sCometTimeTableGrandGalaxy2,
+    sCometTimeTableGrandGalaxy3,
+    sCometTimeTableGrandGalaxy4,
+    sCometTimeTableGrandGalaxy5,
+    sCometTimeTableGrandGalaxy6,
+};
+
+const GalaxyCometSerializeInfo overrideTerraceDomeState = { 0x0007, 0 };
```

Comment:
Even if no official symbol is available, prefix variables with the `static`/`const` qualifiers accessible in a single CPP file with `s` or `c` (preferable `c` if just `const`, and `s` if `static` or `static const`).